EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -3208,35 +3208,35 @@ QUEST_LV_0300_20160811_003207	As you can tell by secret devices, what kind of a
+QUEST_LV_0300_20160811_003216	Demon barriers are protecting King Kadumel from any harm.{nl}That is why we made the Magic Control Scroll.{nl}
+QUEST_LV_0300_20160811_003217	That will suppress the barriers and then you must strike King Kadumel.{nl}However, the Magic Control Scroll will not keep the barriers down completely.{nl}
+QUEST_LV_0300_20160811_003218	It will regenerate again and again.{nl}So, you have to use Magic Control Scroll every time it rises up.{nl}
+QUEST_LV_0300_20160811_003219	Got it?{nl}Now, let's go and defeat King Kadumel.
 QUEST_LV_0300_20160811_003220	King Kadumel is in Offender Institution.{nl}Be careful and keep what I have told you in mind at all time.
 QUEST_LV_0300_20160811_003221	King Kadumel was one of Nebulas??minions.{nl}Demon Lord Nebulas is far stronger than King Kadumel.
 QUEST_LV_0300_20160811_003222	Are you hurt?{nl}
-QUEST_LV_0300_20160811_003223	That?셲 a relief.{nl}Now disabling demon barriers.
+QUEST_LV_0300_20160811_003223	That's a relief.{nl}Now disabling demon barriers.
 QUEST_LV_0300_20160811_003224	To disable demon barrier, you must go down to Supply Room.{nl}The barrier is quite strong and requires something in return. {nl}
-QUEST_LV_0300_20160811_003225	The secret device in the supply room has Orb of Dominance Magic.{nl}I believe it is the last card up King Kadumel?셲 sleeve.{nl}
-QUEST_LV_0300_20160811_003226	Dominance Magic is a powerful magic, it is enough to dispel demon barrier.{nl}You might face a slight problem but first, let?셲 find the orb.
-QUEST_LV_0300_20160811_003227	The secret device in the supply room can only be opened by entering a password.{nl}The problem is the memory about the password, I don?셳 have. {nl}
+QUEST_LV_0300_20160811_003225	The secret device in the supply room has Orb of Dominance Magic.{nl}I believe it is the last card up King Kadumel's sleeve.{nl}
+QUEST_LV_0300_20160811_003226	Dominance Magic is a powerful magic, it is enough to dispel demon barrier.{nl}You might face a slight problem but first, let's find the orb.
+QUEST_LV_0300_20160811_003227	The secret device in the supply room can only be opened by entering a password.{nl}The problem is the memory about the password, I don't have. {nl}
 QUEST_LV_0300_20160811_003228	Perhaps, there might be a part of my soul that remembers the password.. {nl}But at the moment, we have no time, we have to solve it ourselves.{nl}
 QUEST_LV_0300_20160811_003229	The password is comprised of numbers. {nl}There might be some clues around the device.
 QUEST_LV_0300_20160811_003230	I should think about how to use the Orb of Dominance Magic when we acquire it.{nl}It is very powerful. A powerful magic requires something of equal worth in return.
 QUEST_LV_0300_20160811_003231	You got the Orb of Dominance Magic.{nl}Nice job. {nl}
 QUEST_LV_0300_20160811_003232	Also, I figured out a way to use Orb of Dominance Magic.
 QUEST_LV_0300_20160811_003233	I did say something about ?쐅iving something in return?? right?{nl}
```

Comment:
Yeah it's gibberish, it should say 'giving' , i have made a commit to fix it but i'm not able to push commits at the moment until i get home in a few hours. ( i lost net access when attempting to sync the aforementioned corrections )